README.md

```diff
@@ -1,73 +1,161 @@
-heroku config:set OAUTH_CLIENT_SECRET=<client-secret-from-created-app-in-dev-portal>
-heroku config:set OAUTH_URL=https://oauth.onshape.com
-heroku config:set WEBHOOK_CALLBACK_ROOT_URL=https://<url-from-heroku-create.herokuapp.com>
-heroku config:set SESSION_SECRET=<a-cryptographically-secure-string>
-```
-10. You can confirm your configuration settings by running `heroku config`. You should see all of the above.
-11. Commit your (local) configuration changes, and push to Heroku. This will start a build process, after which your application will be up and running.
-`git commit -am "Updated configuration." && git push heroku master`. If you would like to watch the log as the build is running you can run `heroku logs --tail`.
-
-## Usage
-Once your application is deployed and configured, you can subscribe to it through the [Onshape App Store](https://appstore.onshape.com), and add it to your document. You can then use the dropdown menu at the top to select the element to translate and render, and it will be shown in the page.
-
-Note that if you have a complex model with a lot of parts there are two implications: the dropdown list can be quite long; and translating the model to GLTF can be time consuming. If you think there is an issue loading or rendering your model, you can open the Javascript console of your browser to check for any errors.
-
-Once the model is rendered, the following controls are available to you:
-Control | How to Use
-------- | ----------
-Zoom | Mouse wheel
-Rotate | Left-click and move mouse
-Pan | Right-click and move mouse
-
-## References
-* [Onshape Developer Portal](https://dev-portal.onshape.com)
-    * [Help / Documentation](https://dev-portal.onshape.com/help)
-* [Heroku](https://heroku.com)
-    * [Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli)
-* [GLTF information](https://www.khronos.org/gltf/) from Khronos Group
-* [three.js](https://threejs.org/) library to render GLTF
-                     
+## **README**
+The GLTF Viewer is a sample application that demonstrates how to **deploy an Onshape application**. It enables visualization of GLTF data translated from an Onshape model. It is a Node.JS application that runs as a tab inside an Onshape document. Onshape passes the document context to the viewer, which is used to help generate the GLTF visualization. 
+
+This application also demonstrates a **successful authentication process via OAuth2**.
+
+This application is **deployed using Heroku**. You must either have a Heroku account to follow the steps below, or replace the Heroku steps with the deployment platform of your choosing.
+
+## Prerequisites
+
+1. Install the necessary software:  
+    * Git
+    * npm
+    * Heroku CLI (or the deployment server of your choosing)
+      * MacOS: `brew tap heroku/brew && brew install heroku` 
+      * Linux distributions that support Snap applications: `sudo snap install --classic heroku`
+      * Windows: Coming soon
+    * Docker 
```

Comment:
I am not sure we need Docker

Reply:
Same. I got Heroku errors when Docker was closed.